sdk/va-report-components/documentation/docs/api/ObjectHandle.md

```diff
@@ -83,6 +93,7 @@ Adds an event listener to the `ObjectHandle` to call the supplied listener when
 
 `eventType` is a string that represents the event type to listen for. These event types are supported:
 - `"selectionChanged"` for listening for selection changes in the object.
+- `"readyStateChanged"` for listening for changes in the object’s ready state.
```

Comment:
```suggestion
- `"readyStateChanged"` for listening to changes on the `readyState` property.
```

Reply:
Seems better to directly reference the property that this change event is tracking.